site/app/libraries/database/DatabaseQueries.php

```diff
@@ -82,6 +82,14 @@ public function getSubmittyUser(string $user_id): ?User {
         return ($this->submitty_db->getRowCount() > 0) ? new User($this->core, $this->submitty_db->row()) : null;
     }
 
+    /**
+     * Get user from submitty database by user_id
+     */
+    public function getUserBySubmittyId($user_id) {
+        $this->submitty_db->query("SELECT * FROM users WHERE user_id=?", [$user_id]);
+        return ($this->submitty_db->getRowCount() > 0) ? new User($this->core, $this->submitty_db->row()) : null;
+    }
```

Comment:
This function already exists by a different name, make sure you check what already exists when you create a PR.